inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/AnnotationQueueOverviewDataProvider.java

```diff
@@ -113,27 +138,23 @@ else if (getSort().getProperty().equals(headers.get(3))) {
             else if (getSort().getProperty().equals(headers.get(4))) {
                 if (o1.getUpdated() == null) {
                     return dir;
-                }
-                else if (o2.getUpdated() == null) {
+                } else if (o2.getUpdated() == null) {
                     return dir * -1;
-                }
-                else {
+                } else {
                     return dir * (o1.getUpdated().compareTo(o2.getUpdated()));
                 }
-            }
-            else {
+            } else {
                 return 0;
             }
         });
 
 
         //Reset
-        this.shownDocuments.clear();
+        this.shownDocuments = new ArrayList<>();
```

Comment:
Why new list instead of clear? Couldn't the shownDocuments be final?

Reply:
Talked about the issue with Jan, clear() results in an error upon first loading the page, therefore new ArrayList() to avoid. Also, as new ArrayList reallocates memory instead of nullifing the ArrayList contents with clear() I normally prefer this solution.
And as the shown documents may change due to the filtering, I do not think that they can be final.

Reply:
The container (i.e the list) could be final. Note that "final" is not necessarily "immutable". I don't understand why clear() results in an error on the first loading and what this has to do with the list.

Reply:
Changed it to final and initalized it in both constructors, even though one of them didnt need the variable. However, no more nullpointer is created here when using clear()